packages/swirl-components/src/components/flip-dialog/flip-dialog.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Method,
+  Prop,
+  State,
+} from "@stencil/core";
+import A11yDialog from "a11y-dialog";
+import classnames from "classnames";
+
+export type FlipDialogIntent = "primary" | "critical";
+
+/**
+ * @slot slot - The dialog content
+ */
+@Component({
+  shadow: true,
+  styleUrl: "flip-dialog.css",
+  tag: "flip-dialog",
+})
+export class FlipDialog {
+  @Prop() hideLabel?: boolean;
+  @Prop() intent?: FlipDialogIntent = "primary";
+  @Prop() label!: string;
+  @Prop() primaryActionLabel?: string;
+  @Prop() secondaryActionLabel?: string;
+
```

Comment:
isn't the vertical-button-group option missing?
![image](https://user-images.githubusercontent.com/23700195/189900228-d94127d0-98c5-4e81-a8e2-5f2c226b5d39.png)

Reply:
Should wrap automatically, when one of the button labels gets too long.

Reply:
shouldn't be there a programmatic option through a prop? at least i interpret the design in that way 😇